### PR TITLE
Increases restart/shuttle call vote delay

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
tweak: Increases restart vote delay from 30 to 35 mins
tweak: Shuttle vote now makes the shuttle call normally
/:cl:

## About The Pull Request
Title. People complain that these votes are used too early into the round and I agree myself.

## Why It's Good For The Game
No more forced 5 min shuttle call.

### CODE REVIEW

**Comment by ghost** [comment]
-1

**Comment by yoyobatty** [comment]
I knew you would downvote this, because you have a huge habit of voting to call the shuttle way too early.

**Comment by mkikt4743** [comment]
this is more of an Enethan and Steamport problem than a code problem, but still +1

**Comment by Toxici11i** [comment]
+1, nerfs crew and antags by forcing them to co-exist. shuttle vote being 5 minutes is really gay and is abused by people joining, seeing the round is over X minutes, and initiating a vote

**Comment by diegoxt45** [comment]
-1

**Comment by yoyobatty** [comment]
> -1

not a designer, diego

**Comment by diegoxt45** [comment]
> > -1
> 
> not a designer, diego

stay mad

**Comment by diegoxt45** [comment]
this is terrible for multiple  reasons but i can't actually be bothered to say why


i said why in the discord

**Comment by yoyobatty** [comment]
Changed delay to 35 mins, restart delay back to 60 mins. Shuttle call vote success doesn't make shuttle come in 5 minutes but standard 10. Happy?

**Comment by diegoxt45** [comment]
> Changed delay to 35 mins, restart delay back to 60 mins. Shuttle call vote success doesn't make shuttle come in 5 minutes but standard 10. Happy?

can it still be recalled though?

**Comment by yoyobatty** [comment]
Yes

**Comment by diegoxt45** [comment]
kinda gay. but i'l settle for it

File: hippiestation/code/controllers/subsystem/vote.dm [hunk 1]
```diff
@@ -236,11 +236,11 @@ datum/controller/subsystem/vote
 		var/shuttle_timer = SSshuttle.emergency.timeLeft()
 		if(shuttle_timer >= 300 || (SSshuttle.emergency.mode != SHUTTLE_CALL && SSshuttle.emergency.mode != SHUTTLE_DOCKED && SSshuttle.emergency.mode != SHUTTLE_ESCAPE)) // hippie -- fix shuttle votes upping timers to 5 minutes
 			if(SSshuttle.emergency.mode == SHUTTLE_CALL && shuttle_timer >= 300)	//Apparently doing the emergency request twice cancels the call so these check are just in case
-				SSshuttle.emergency.setTimer(3000)
+				SSshuttle.emergency.setTimer(6000)
 				priority_announce("The emergency shuttle will arrive in [SSshuttle.emergency.timeLeft()/60] minutes.")
 			else if (SSshuttle.emergency.mode != SHUTTLE_CALL)
 				SSshuttle.emergency.request()
-				SSshuttle.emergency.setTimer(3000)
+				SSshuttle.emergency.setTimer(6000)
```
Comment (ghost) [inline hippiestation/code/controllers/subsystem/vote.dm:243]: you might wanna set noRecall to true, else an antag might murderbone the station and recall any votes

**Comment by yoyobatty** [comment]
Okay, I set no recall to true.

**Comment by Maroonenahmoron** [comment]
+1, it might take more than 5 minutes for people to finish up what they're doing.

**Review by ghost — APPROVED** [review]

**Pull request merged**